### DISCUSSION TITLE
When installing the source, this problem appears.  Use Ubuntu 18.4

### DISCUSSION THREAD

**Discussion by TH3CZAR-VIP** [python-telegram-bot/python-telegram-bot#3166] [Q&A]
![IMG_20220724_002010_626](https://user-images.githubusercontent.com/109393782/180623232-779551da-929d-41a9-94f0-aa8cea418fc3.jpg)

**Comment by harshil21** [comment]
Hello, this library supports only Python 3.7+. You'll have to upgrade your python version to use this library.

**Comment by Bibo-Joshi** [comment]
Unless you have a very specific reason to install from source, I highly recommend to install PTB from pypi.org instead of building it from source. In any case, the latest PTB versions support only python 3.7+ and you're apparently running python 3.5-
